packages/synthetics-sdk-broken-links/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@google-cloud/synthetics-sdk-api": "google-cloud-synthetics-sdk-api-0.1.0.tgz",
```

Comment:
side node - can this be updated to target an actual version of this dependency now?

Reply:
All broken link synthetic api information is in broken_links_final. It has not been pushed to main / npm so the dependency will stay as a tar file until we merge into main.

Reply:
got it, thanks!